src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.tasks.Task;
+import duke.tasks.taskType;
+import duke.ui.Ui;
+
+import java.util.ArrayList;
+
+public class FindCommand extends Command{
```

Comment:
Minor suggestion: Maybe `CommandFinder` instead of `FindCommand` since 'Find' in this case is a verb?

From [guideline on naming](https://se-education.org/guides/conventions/java/intermediate.html#naming):
> Class/enum names must be nouns and written in PascalCase.